RecoBTag/TrackProbability/test/CalibrationSkeleton.cc

```diff
@@ -81,93 +77,83 @@ using namespace reco;
 //
 
 class CalibrationSkeleton : public edm::EDAnalyzer {
-   public:
+public:
   explicit CalibrationSkeleton(const edm::ParameterSet&);
 
-  virtual void beginJob()
-  {
-    bool resetData=true;
-    bool newBinning=false;
+  virtual void beginJob() {
+    bool resetData = true;
+    bool newBinning = false;
     edm::FileInPath f2d("RecoBTag/TrackProbability/data/2DHisto.xml");
     edm::FileInPath f3d("RecoBTag/TrackProbability/data/3DHisto.xml");
-    calibrationNew   =  new AlgorithmCalibration<TrackClassFilterCategory,CalibratedHistogramXML>((f3d.fullPath()).c_str());
-    calibration2dNew =  new AlgorithmCalibration<TrackClassFilterCategory,CalibratedHistogramXML>((f2d.fullPath()).c_str());
-    if(resetData)
-      {
-	vector<pair<TrackClassFilterCategory, CalibratedHistogramXML> > data = calibrationNew->categoriesWithData();
-	vector<pair<TrackClassFilterCategory, CalibratedHistogramXML> > data2d = calibration2dNew->categoriesWithData();
-	for(unsigned int i = 0 ; i < data.size();i++)
-          {
-            if(newBinning) data[i].second = CalibratedHistogram(1000, 0, 50);
-            else           data[i].second = CalibratedHistogram();
-          }
-	for(unsigned int i = 0 ; i < data2d.size();i++)
-          {
-            if(newBinning) data2d[i].second = CalibratedHistogram(1000, 0, 50);
-            else           data2d[i].second = CalibratedHistogram();
-          }
-	
+    calibrationNew =
```

Comment:
I dont see where `calibrationNew` is deleted. But this file is under \test so fine...